tests/molecule/requirements.in

```diff
@@ -0,0 +1,7 @@
+# Can't use `-r../requirements.txt` because that pins package versions, e.g. for
```

Comment:
I'd rather keep this in `tests/requirements.in`. The corresponding venv can be shared between various tox targets in its configuration.